### PR TITLE
Updated Java example for Attested Data

### PR DESCRIPTION
**Pull request by rifrankl**

### CODE REVIEW

File: imdssample.java [hunk 1]
```diff
@@ -1,40 +1,172 @@
...
+	{
+		return QueryImds(InstanceEndpoint, "2017-04-02");		
+	}
 	
-	//imds url   
-	String imdsUrl= "http://169.254.169.254/metadata/instance?api-version=2017-04-02";
+	private static String QueryAttestedEndpoint()
+    {
+		String nonce = "nonce=" + NonceValue;
+        return QueryImds(AttestedEndpoint, "2019-04-30", nonce);
```
Comment (KumariSupriya) [inline imdssample.java]: since we are already updating the code ,should we update the api version too

File: imdssample.java [hunk 2]
```diff
@@ -1,40 +1,172 @@
...
+		}
+		catch(Exception ex)
+		{
+			System.out.println("Exception validating data: " + ex.getMessage());
+		}
+	}
+	
+	private static String QueryInstanceEndpoint()
+	{
+		return QueryImds(InstanceEndpoint, "2017-04-02");
```
Comment (KumariSupriya) [inline imdssample.java]: since we are already updating the example ,should we update the api-version  for instance too ?

Comment (rifrankl) [reply imdssample.java]: Done. We also probably should update this in the other examples, as well. I'll do that in a separate PR.

File: imdssample.java [hunk 3]
```diff
@@ -1,40 +1,172 @@
...
+	
+	private static void ValidateAttestedData(byte[] decoded)
+	{
+		try
+		{
+			CMSSignedData signature = new CMSSignedData(decoded);
+			CMSTypedData signedData = signature.getSignedContent();
+			String signedDataString = new String((byte[])signedData.getContent());
+			System.out.println("Attested data: " + signedDataString);
+			AttestedData data = new Gson().fromJson(signedDataString, AttestedData.class);
```
Comment (KumariSupriya) [inline imdssample.java]: good to add comments here to verify the plan info and subscription id too

**Comment by KumariSupriya** [comment]
#sign-off

**Pull request merged**